pyartcd/pyartcd/__main__.py

```diff
@@ -4,7 +4,7 @@
 from pyartcd.pipelines import (
     build_microshift, check_bugs, gen_assembly, ocp4_konflux, prepare_release, promote, rebuild,
     review_cvp, tarball_sources, build_sync, build_rhcos, ocp4_scan, images_health, operator_sdk_sync,
-    olm_bundle, ocp4, scan_for_kernel_bugs, tag_rpms, advisory_drop, cleanup_locks, brew_scan_osh, sigstore_sign, update_golang, rebuild_golang_rpms, scan_fips
+    olm_bundle, ocp4, scan_for_kernel_bugs, tag_rpms, advisory_drop, cleanup_locks, brew_scan_osh, sigstore_sign, update_golang, rebuild_golang_rpms, scan_fips, doomsday
```

Comment:
Instead of naming it `doomsday` I would recommend using `quay_doomsday_backup` or something like that

Reply:
I renamed it to `quay_doomsday_backup` as you suggested